src/main/java/by/yahor_kulesh/utils/ObjectSetAsMap.java

```diff
@@ -0,0 +1,38 @@
+package by.yahor_kulesh.utils;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.function.Consumer;
+
+public class ObjectSetAsMap {
```

Comment:
ObjectSetAsArray is enough, you can remove this one (ObjectSetAsMap) as redundant

Reply:
dropped